Encoders/SFBCoreAudioEncoder.mm

```diff
@@ -438,7 +438,7 @@ - (BOOL)openReturningError:(NSError **)error
 		}
 
 		if(audioConverter) {
-			for(NSNumber *key in audioConverterPropertySettings) {
+			for(NSNumber *key in audioConverterPropertySettings.allKeys) {
```

Comment:
This change is not necessary. Fast enumeration for NSDictionary iterates over the keys: https://developer.apple.com/documentation/foundation/nsdictionary?language=objc

Reply:
thank you, i get it.